docs/source/access/accessing_cod.ipynb

```diff
@@ -0,0 +1,525 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Accessing Cloud Optimized Data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The following is an example that uses Shuttle Radar Topography Mission (SRTM) Cloud Optimized Geotiff data from the MAAP data store, via MAAP CMR API search. In this example, we read in elevation data using a bounding box tile.\n",
+    "\n",
+    "First we install any necessary packages. Please note that the following block of code only needs to be run if needed within a new workspace and that you may need to restart the kernel to use updated packages."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Collecting package metadata (current_repodata.json): done\n",
+      "Solving environment: done\n",
+      "\n",
+      "# All requested packages already installed.\n",
+      "\n",
+      "\n",
+      "Note: you may need to restart the kernel to use updated packages.\n"
+     ]
+    }
+   ],
+   "source": [
+    "# only run this block if needed in a new workspace\n",
+    "%conda install -y -c conda-forge geopandas shapely folium rasterio pygeos rio-cogeo"
```

Comment:
I had to use pip, I replaced this line with:

```
!pip install geopandas shapely folium rasterio pygeos rio-cogeo
```

Reply:
That's not good, going forward the official recommendation from JPL is always defer to conda first. Can you restart your workspace, that should clear any previously installed conflicting packages.

Reply:
Replaced with:
%conda install -y -c conda-forge shapely folium pygeos
!pip install -U geopandas rasterio>=1.2.3 rio-cogeo